### PR TITLE
all data uploaded to database

### PR DESCRIPTION
**Pull request by vpatel89**
@sbirvin1s @MonicaCupp

### CODE REVIEW

**Review by MonicaCupp — APPROVED** [review]
Yeah, this looks good to me! Great work!

File: server/schema.sql [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  reviews(
+    id SERIAL PRIMARY KEY NOT NULL,
+    product_id SERIAL NOT NULL,
+    rating smallint,
+    date bigint NOT NULL,
+    summary TEXT,
+    body TEXT,
+    recommend boolean,
+    reported boolean,
+    reviewer_name VARCHAR(255),
```
Comment (sbirvin1s) [inline server/schema.sql:16]: Was there a reason you went with `VARCHAHR(255)` here instead of just using TEXT?

File: server/schema.sql [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    helpfulness INTEGER
+);
+
+\copy reviews FROM 'server/data/reviews.csv' DELIMITER ',' CSV HEADER;
+
+
+
+CREATE TABLE
+  review_photos(
+    id SERIAL PRIMARY KEY NOT NULL,
```
Comment (sbirvin1s) [inline server/schema.sql:29]: More preference than anything, but if you are using `SERIAL` you can likely drop the `NOT NULL` condition from the schema.

According to [Postgres Docs](https://www.postgresql.org/docs/14/datatype-numeric.html#DATATYPE-SERIAL) it appears that `SERIAL` has that condition included by default

File: server/schema.sql [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    body TEXT,
+    recommend boolean,
+    reported boolean,
+    reviewer_name VARCHAR(255),
+    reviewer_email VARCHAR(255),
+    response TEXT,
+    helpfulness INTEGER
+);
+
+\copy reviews FROM 'server/data/reviews.csv' DELIMITER ',' CSV HEADER;
```
Comment (sbirvin1s) [inline server/schema.sql:23]: Nice work adding the csv to the server file 👍

**Review by sbirvin1s — APPROVED** [review]
Few minor comments that you might look into, but everything looks good 👍

**Pull request merged**